diff -Naurp nagios-3.1.0/html/includes/utils.inc.php nagios-3.1.0.oden/html/includes/utils.inc.php
--- nagios-3.1.0/html/includes/utils.inc.php	2008-12-14 15:52:23.000000000 +0100
+++ nagios-3.1.0.oden/html/includes/utils.inc.php	2009-01-25 18:48:27.000000000 +0100
@@ -9,7 +9,7 @@ function get_update_information(){
 	// initialize array
 	$updateinfo=array(
 		"found_update_info" => false,
-		"update_checks_enabled" => true,
+		"update_checks_enabled" => false,
 		"last_update_check" => "",
 		"update_available" => false,
 		"update_version" => "",
diff -Naurp nagios-3.1.0/include/nagios.h nagios-3.1.0.oden/include/nagios.h
--- nagios-3.1.0/include/nagios.h	2008-12-14 15:52:23.000000000 +0100
+++ nagios-3.1.0.oden/include/nagios.h	2009-01-25 18:45:43.000000000 +0100
@@ -125,7 +125,7 @@ extern "C" { 
 
 #define DEFAULT_ADDITIONAL_FRESHNESS_LATENCY			15	/* seconds to be added to freshness thresholds when automatically calculated by Nagios */
 
-#define DEFAULT_CHECK_FOR_UPDATES                               1       /* should we check for new Nagios releases? */
+#define DEFAULT_CHECK_FOR_UPDATES                               0       /* should we check for new Nagios releases? no, we shouldn't... */
 #define DEFAULT_BARE_UPDATE_CHECK                               0       /* report current version and new installs */
 #define MINIMUM_UPDATE_CHECK_INTERVAL                           60*60*22 /* 22 hours minimum between checks - please be kind to our servers! */
 #define BASE_UPDATE_CHECK_INTERVAL                              60*60*22 /* 22 hours base interval */
diff -Naurp nagios-3.1.0/sample-config/nagios.cfg.in nagios-3.1.0.oden/sample-config/nagios.cfg.in
--- nagios-3.1.0/sample-config/nagios.cfg.in	2009-01-25 18:48:53.000000000 +0100
+++ nagios-3.1.0.oden/sample-config/nagios.cfg.in	2009-01-25 18:46:21.000000000 +0100
@@ -729,7 +729,7 @@ interval_length=60
 # by Nagios Enterprises from the update check is processed in accordance 
 # with our privacy policy - see http://api.nagios.org for details.
 
-check_for_updates=1
+check_for_updates=0
 
 
 
