diff -Naur -x svn-commit.tmp~ nagios/sample-config/cgi.cfg.in nagios-mdv-config/sample-config/cgi.cfg.in
--- nagios/sample-config/cgi.cfg.in	2012-02-13 21:40:42.000000000 +0100
+++ nagios-mdv-config/sample-config/cgi.cfg.in	2013-02-03 13:28:10.283047088 +0200
@@ -116,7 +116,7 @@
 # not use authorization.  You may use an asterisk (*) to
 # authorize any user who has authenticated to the web server.
 
-authorized_for_system_information=nagiosadmin
+authorized_for_system_information=nagios
 
 
 
@@ -128,7 +128,7 @@
 # an asterisk (*) to authorize any user who has authenticated
 # to the web server.
 
-authorized_for_configuration_information=nagiosadmin
+authorized_for_configuration_information=nagios
 
 
 
@@ -141,7 +141,7 @@
 # You may use an asterisk (*) to authorize any user who has
 # authenticated to the web server.
 
-authorized_for_system_commands=nagiosadmin
+authorized_for_system_commands=nagios
 
 
 
@@ -154,8 +154,8 @@
 # to authorize any user who has authenticated to the web server.
 
 
-authorized_for_all_services=nagiosadmin
-authorized_for_all_hosts=nagiosadmin
+authorized_for_all_services=nagios
+authorized_for_all_hosts=nagios
 
 
 
@@ -168,8 +168,8 @@
 # authorization).  You may use an asterisk (*) to authorize any
 # user who has authenticated to the web server.
 
-authorized_for_all_service_commands=nagiosadmin
-authorized_for_all_host_commands=nagiosadmin
+authorized_for_all_service_commands=nagios
+authorized_for_all_host_commands=nagios
 
 
 
diff -Naur -x svn-commit.tmp~ nagios/sample-config/nagios.cfg.in nagios-mdv-config/sample-config/nagios.cfg.in
--- nagios/sample-config/nagios.cfg.in	2012-04-23 13:01:57.000000000 +0200
+++ nagios-mdv-config/sample-config/nagios.cfg.in	2013-02-03 16:23:34.067118252 +0200
@@ -54,6 +54,8 @@
 #cfg_dir=@sysconfdir@/switches
 #cfg_dir=@sysconfdir@/routers
 
+# Definitions for plugin commands
+cfg_dir=@sysconfdir@/plugins.d
 
 
 
@@ -728,7 +730,7 @@
 # by Nagios Enterprises from the update check is processed in accordance 
 # with our privacy policy - see http://api.nagios.org for details.
 
-check_for_updates=1
+check_for_updates=0
 
 
 
